pages/consoles.html

```diff
@@ -0,0 +1,418 @@
+---
+permalink: /consoles/index.html
+title: "Console Support - Godot Engine"
```

Comment:
```suggestion
title: "Console Support"
```
"- Godot Engine" is added automatically, as far as I know.

EDIT: I think I was wrong about the capitalization, so changed back!